cg/meta/backup.py

```diff
@@ -12,17 +12,18 @@
 class BackupApi:
     """ Class for retrieving FCs from backup """
 
-    def __init__(self, status: Store, pdc_api: PdcApi):
+    def __init__(self, status: Store, pdc_api: PdcApi, max_flowcells: int):
         self.status = status
         self.pdc = pdc_api
+        self.max_flowcells = max_flowcells
 
-    def maximum_flowcells_ondisk(self, max_flowcells: int = 1000) -> bool:
+    def maximum_flowcells_ondisk(self) -> bool:
         """Check if there's too many flowcells already "ondisk"."""
         ondisk_flowcells = self.status.flowcells(status="ondisk").count()
         LOG.debug("ondisk flowcells: %s", ondisk_flowcells)
-        return ondisk_flowcells > max_flowcells
+        return ondisk_flowcells > self.max_flowcells
 
-    def check_processing(self, max_flowcells: int = 3) -> bool:
+    def check_processing(self, max_flowcells: int = 1) -> bool:
```

Comment:
This is confusing, are we using the same variable name with a different meaning here? Or why not use `self.max_flowcells`?

Reply:
I'm not sure, this is not my code. I only changed the number from 1000 to 1250 :)

You have a good point of course and I'll have a look.

Reply:
Answer: we're dealing with two different max_flowcells! one for the max number on disk, and on for the max number being processed. https://github.com/Clinical-Genomics/cg/pull/397/commits/eda51b768f6bc26eac7149720c70ba096a80d29e will make it more clear which is which :)